### PR TITLE
Thefeminineurgetodropout

### PR DESCRIPTION
**Pull request by ameliarwig**

### CODE REVIEW

File: core/src/com/mygdx/game/Components/Renderable.java [hunk 1]
```diff
@@ -50,6 +51,14 @@ public Renderable(int atlasId, String texName, RenderLayer layer) {
         RenderingManager.addItem(this, layer);
     }
 
+    public Renderable(String img, RenderLayer layer) {
+        this();
+        Texture texture = new Texture(img);
```
Comment (printincident) [inline core/src/com/mygdx/game/Components/Renderable.java:56]: Was this not already in there ??

File: core/src/com/mygdx/game/Entitys/Monster.java [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    private String getDirection(Vector2 dir) {
+        if (!currentDir.equals(dir) && directions.containsKey(dir)) {
+            currentDir.set(dir);
+            return directions.get(dir);
+        }
+        return "";
+    }
+
+    /**
+     * will rotate the ship to face the direction (just changes the sprite doesn't actually rotate)
```
Comment (printincident) [inline core/src/com/mygdx/game/Entitys/Monster.java:72]: Remove some of these comments that dont make sense

File: tests/src/WeatherTests.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertTrue;
+
+@RunWith(GdxTestRunner.class)
+public class WeatherTests {
+
+    @Test
+    public void testTrue() {
```
Comment (printincident) [inline tests/src/WeatherTests.java:11]: Great tests

Comment (ameliarwig) [reply tests/src/WeatherTests.java:11]: theyre in the wrong pr ahah

Comment (printincident) [reply tests/src/WeatherTests.java:11]: ok remove it

File: core/src/com/mygdx/game/Entitys/Monster.java (hunk 2 above)
Comment (ameliarwig) [reply core/src/com/mygdx/game/Entitys/Monster.java:72]: :0

**Review by printincident — APPROVED** [review]

**Pull request merged**